COLLADAStreamWriter/include/COLLADASWTechnique.h

```diff
@@ -49,12 +49,19 @@ namespace COLLADASW
 
         /** Adds a parameter to the technique */
 		void addParameter(const String &paramName, const int &value, const String &sid = "", const String &type = "", TagType tagType = CUSTOM);
+        void addParameter(const String &paramName, const int &value0, const int &value1, const String &sid = "", const String &type = "", TagType tagType = CUSTOM);
```

Comment:
What is the schema for these so called "parameters"?

This class appears to be evolving from a simple opaque preservation of unparsed data into now a hard coding of some implicit data model into the technique element, which is a false assertion.

The technique element may provide it's own schema via the xmlns attribute for use by a validating XML parser, otherwise it's simply a bag of elements (as implemented here which is okay). I see no support for xmlns in the implementation thus far. I think this entire functionality would be better implemented by enhancing addChildElement() and closeChildElement() in line with the specification. At best these parameter functions are convenience functions for a popular use-case. The technique element is by no means limited to this functionality (it's actually a quite powerful declaration).